core/lib/soci_postgresql/session.cpp

```diff
@@ -128,3 +128,14 @@ postgresql_blob_backend * postgresql_session_backend::make_blob_backend()
 {
     return new postgresql_blob_backend(*this);
 }
+
+bool postgresql_session_backend::is_alive() const {
+    auto res = PQexec(conn_, "SELECT 1");

```

Comment:
Add `const` qualifier to that variable

Reply:
I can’t, because `PQresult` expects a `PQconn*`. I don’t want `const_cast` here, especially because `res` won’t be used anymore after that call.